doc/Tutorials.xml

```diff
@@ -313,6 +313,97 @@ AutoDoc( rec( gapdoc := rec( gap_root_relative_path := true ) ) );
 
 </Subsection>
 
+<Subsection Label="Tut:Checklist">
+<Heading>Checklist for converting an existing manual to use &AutoDoc;</Heading>
+
+Here is a checklist for authors of a package &PackageName;, 
+which already has a manual, who wish to use &AutoDoc; 
+to build the manual from now on. 
+We assume that the manual is currently built by reading a file 
+<File>makedoc.g</File> and that the main <File>.xml</File> file 
+is named <File> manual.xml</File>. 
+<P/> 
+The files <File>PackageInfo.g</File>, <File>makedoc.g</File>, 
+<File>doc/title.xml</File> and <File>doc/PackageName.xml</File> 
+(if it exists) will be altered by this procedure, 
+so it may be wise to keep backup copies. 
+<P/> 
+You should have copies of the &AutoDoc; files <File>PackageIngo.g</File> 
+and <File>makedoc.g</File> to hand when reading these instructions. 
+<P/> 
+<List>
+<Item> 
+Copy <File>AutoDoc/makedoc.g</File> to <File>PackageName/makedoc.g</File>. 
+</Item>
+<Item>
+Edit <File>PackageName/makedoc.g</File> as follows. 
+  <List>
+  <Item> 
+  Change the header to anything suitable. 
+  </Item> 
+  <Item> 
+  Replace <Code>LoadPackage("AutoDoc");</Code> 
+  with <Code>LoadPackage("PackageName");</Code>. 
+  </Item> 
```

Comment:
I think `LoadPackage("PackageName");` should be added, as you need to load AutoDoc for the AutoDoc  command.

Reply:
Both now loaded